src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/OpenSslPkcs12Reader.cs

```diff
@@ -24,32 +24,24 @@ private OpenSslPkcs12Reader(SafePkcs12Handle pkcs12Handle)
 
         public static bool TryRead(byte[] data, out OpenSslPkcs12Reader pkcs12Reader)
         {
-            SafePkcs12Handle handle = Interop.Crypto.DecodePkcs12(data, data.Length);
-
-            if (!handle.IsInvalid)
-            {
-                pkcs12Reader = new OpenSslPkcs12Reader(handle);
-                return true;
-            }
+            Exception ignored;
+            return TryRead(data, out pkcs12Reader, out ignored, captureException: false);
```

Comment:
Nit: this can be:
```C#
public static bool TryRead(byte[] data, out OpenSslPkcs12Reader pkcs12Reader) =>
    TryRead(data, out pkcs12Reader, out _, captureException: false);
```